### ISSUE TITLE
swCSTp(..., eos="gsw") build tests fail

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#746]
In ``oce/tests/testthat/test_sw.R`` the fourth test was failing without setting a KLUDGE value of 1e-4, which is far beyond computational uncertainty and thus suggests a problem. Clark and I have been discussing this at the bottom of comments for issue #741 but I opened this new one because the older one was meandering all across topic space.
```R
# 13. electrical conductivity
expect_equal(swCSTp(35, T90fromT68(15), 0, eos="unesco"),  1)
expect_equal(swSCTp( 1, T90fromT68(15), 0, eos="unesco"), 35)
expect_equal(swCSTp(35,            15,  0, eos="gsw"),     1, tolerance=KLUDGE)
expect_equal(swSCTp( 1,            15,  0, eos="gsw"),    35)
```
The failure came about because of a change recently made that relates to the computation of conductivity *ratio* from conductivity itself. See https://github.com/dankelley/oce/issues/741#issuecomment-138624383 for a note on that, and see comments near the end of #741 for discussion, the upshot of which is that we need to do some reading to think of the proper scheme. Since the TEOS-10 documents are so carefully thought-out, my guess is that they will be a good source.

In the meantime, this ``KLUDGE`` factor will remain in the test code. But it will be removed later, since agreement to 0.01 percent is insufficient to indicate clear thought.

**Comment by dankelley** [comment]
I'm coming around to agreeing with Clark that conductivity ratio should be the actual conductivity divided by a *constant*, not divided by a function call whose value may change with various manifestations of that function.  This is consistent with the TEOS-10 (v3.05) fortran code, e.g. in its ``toolbox/gsw_sp_from_c.f90`` code, lines 46 et seq read as follows.
```fortran
! The dimensionless conductivity ratio, R, is the conductivity input, C,
! divided by the present estimate of C(SP=35, t_68=15, p=0) which is 
! 42.9140 mS/cm (=4.29140 S/m), (Culkin and Smith, 1980). 

r = c/gsw_c3515
```
and line 53 of ``modules/gsw_mod_teos10_constants.f90`` reads
```fortran
real (r8), parameter :: gsw_c3515 = 42.9140_r8
```

**Comment by dankelley** [comment]
I've fixed this in ``develop`` commit ce9ca1035fac09b11766cd81392f83d202ee37c1 by simply removing the wrong-headed tests. I'm going to close this, with apologies to Clark, if he is getting ping after ping on his phone for these gh comments lately :smiling_imp:
